assets/app/styles/_mixins.less

```diff
@@ -7,6 +7,19 @@
 }
 
 
+.osc-object-highlight() {
+    &.osc-object {
```

Comment:
@sg00dwin this is exactly what I was envisioning for osc-object styles... just an indicator of clickability and a hover/highlight color indicator... no layout opinions at all